### DISCUSSION TITLE
Installation of DP Gen offline package

### DISCUSSION THREAD

**Discussion by lammps-vasp** [deepmodeling/dpgen#374] [Q&A] [answered]
dpgen -h
bash: dpgen: command not found...

export PATH=$home/.local/bin:$PATH

**Comment by amcadmus** [comment]
Please tell us how did you install the `dpgen` package

**Comment by lammps-vasp** [comment, marked as answer]
It's installed. The path is misspelled
Thanks




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "deepmodeling/dpgen"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2021年4月22日(星期四) 中午12:03
***@***.***&gt;;
抄送:&nbsp;"Zhifeng ***@***.******@***.***&gt;;
主题:&nbsp;Re: [deepmodeling/dpgen] Installation of DP Gen offline package (#374)





 
Please tell us how did you install the dpgen package
